src/content/reference/okteto-cli.mdx

```diff
@@ -782,6 +782,8 @@ Therefore, the development deployment uses the same service account, environment
 | _--reset_     |  (bool)  | Resets the file synchronization service. Use it if the file synchronization service stops working.                     | |
 | _--command_   |  (list)  | The command to execute in the dev container. This overrides the command specified in the `dev` section of the manifest | |
 
+> When employing the `--command` flag, ensure that each command is delineated separately by using `--command`. For example, `okteto up --command bash --command -c --command npm --command start`.
```

Comment:
```suggestion
> When employing the `--command` flag, ensure that each command is delineated separately by using `--command`. For example, `okteto up --command bash --command -c --command npm --command start`.
```
Can we add it in the row of `--command` of the table above? Also, I would simplify the sample to:
```
okteto up --command npm --command start
```